hack/commit-msg-verify.sh

```diff
@@ -1,21 +1,42 @@
 #!/usr/bin/env bash
 
+# This regex matches the pull number with an author from the job spec. Prow does not have a PULL_AUTHOR job var
+export PULL_AUTHOR=$(echo $JOB_SPEC | grep -Po "$PULL_NUMBER,\"author\":\"\K[^\"]*")
+
 # PULL_BASE_SHA and PULL_PULL_SHA are set in prow job
 PULL_BASE_SHA="${PULL_BASE_SHA:-$(git merge-base master HEAD)}"
 PULL_PULL_SHA="${PULL_PULL_SHA:-HEAD}"
+IFS=$'\n' commit_messages=($(git log --pretty=format:"%s" $PULL_BASE_SHA...$PULL_PULL_SHA))
 
-pattern="^[A-Z]+-[0-9]+ \| (feat|fix|docs|style|refactor|test|chore|build|ci|perf): .*$"
-
-commits=$(git rev-list --no-merges $PULL_BASE_SHA..$PULL_PULL_SHA)
-
-for sha in $commits; do
-  message=$(git log --format=%B -n 1 $sha)
-  if ! [[ $message =~ $pattern ]]; then
-    echo "Invalid commit message: $message"
-    echo "Expected format: JIRA_TICKET | TYPE: MESSAGE"
-    echo "Please check CONTRIBUTING.md for more details"
+# Lists of GitHub users for whom the commit validation should be skipped:
+# # This contains the bots used in the ROSA project
+# * openshift-merge-bot[bot] - bot responsible for merigng MRs
+# * openshift-ci[bot] - bot responsible for running tests / approvals
+# * codecov[bot] - bot responsible for reporting code coverage
+declare -a skip_pr_authors=(
+  "openshift-ci[bot]"
+  "openshift-merge-bot[bot]"
+  "codecov[bot]"
```

Comment:
@hunterkepley Codecov doesn't/shouldn't have commit permissions on our repo. We should fail if it does, so I'd suggest removing this author from the list.

cheers,

Rob